docs/content/migration.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Migrate from Primer CSS
+---
+
+Many Primer CSS use cases are supported by Primer ViewComponents.
+
+When migrating from Primer CSS classes to ViewComponents, use this mapping to
+help guide your implementation. This list includes components currently in
+stable status.
+
+| Primer CSS Class | ViewComponent |
+|------------------|---------------|
+| [`State`](https://primer.style/css/components/labels#states)             | [`Primer::StateComponent`](https://primer.style/view-components/components/state)              |
+| [`breadcrumb-item`](https://primer.style/css/components/breadcrumb)      | [`Primer::BreadcrumbComponent`](https://primer.style/view-components/components/breadcrumb)    |
+| [`Counter`](https://primer.style/css/stickersheet/labels#counters)       | [`Primer::CounterComponent`](https://primer.style/view-components/components/counter)          |
+| [`Subhead`](https://primer.style/css/components/subhead)                 | [`Primer::SubheadComponent`](https://primer.style/view-components/components/subhead)          |
+| [`blankslate`](https://primer.style/css/components/blankslate)           | [`Primer::BlankslateComponent`](https://primer.style/view-components/components/blankslate)    |
```

Comment:
could we auto-generate this somehow? 🤔

Reply:
Probably but I'd love to start with something manual and then try that ;)